swagger/src/main/resources/application-swagger.yml

```diff
@@ -7,11 +7,12 @@ gnirps:
             version: ${SWAGGER_VERSION:missing-property:gnirps.common.version}
             license: ${SWAGGER_LICENSE:License:MIT}
             license-url: ${SWAGGER_LICENSE_URL:https://matias.ma/nsfw/}
-            terms-of-service: ${SWAGGER_TITLE:https://matias.ma/nsfw/}
+            terms-of-service: ${SWAGGER_TERMS_OF_SERVICE:https://matias.ma/nsfw/}
+            allowed-paths: ${SWAGGER_ALLOWED_PATHS:/**}
```

Comment:
Well, there could be several ways to make it easy to understand (at least for the 1% of people who read READMEs):
- only expose endpoints matching the `exposed-endpoints` property, with the exception of those matching the `hidden-endpoints` property: say, for instance, expose everything under "_/this_" and "_/that_", but filter out "_/this/private_" and "_/that/as/well_".
- only allow one property to work at once (define a priority order and issue a warning if both are filled): say, for instance, that `exposed-endpoints` is less likely to expose sensitive endpoints by mistake, and make it the only active property when both are filled  (but still issue a warning for the 2% of people who read logs).
- another possibility which you might think of: I was actually hoping you guys would think (rather than me) here!